### DISCUSSION TITLE
请求适配器使用 GlobalFetch 时, GET POST 请求是不是没法设置 responseType

### DISCUSSION THREAD

**Discussion by eivmosn** [alovajs/alova#203] [Q&A]
### 需要新特性解决什么问题？

在做下载附件的功能时 需要指定返回 的数据类型为 Blob 

### 它对你有多重要？

有就更好了

**Comment by JOU-amjs** [comment]
问题性帖子请在discussion中提问
GlobalFetch是封装fetch api实现的，因此也就没有responseType参数，你可以在响应中获得Response，通过Response.blob()获得blob对象

**Reply by eivmosn** [reply]
好的, 我换了 axios 适配器
